pom.xml

```diff
@@ -194,7 +194,7 @@
         <dice-provider-artifact-id>dice-provider</dice-provider-artifact-id>
         <x509-provider-artifact-id>x509-provider</x509-provider-artifact-id>
 
-        <iot-device-client-version>2.1.4</iot-device-client-version>
+        <iot-device-client-version>2.1.5</iot-device-client-version>
         <iot-service-client-version>2.1.6</iot-service-client-version>
```

Comment:
Do we need to remove this from pom given that the service client has been handed off to the service team?

Reply:
I think calling out we no longer have ownership of the service client would make sense.

Reply:
We still use this field to determine what version of the service client we use in our e2e tests (for invoking direct methods on devices, for instance). I'd keep it because of that, but It may be worthwhile to leave a comment here stating that this is now a dependency, not part of this library